src/config/xorg/index.md

```diff
@@ -51,3 +51,35 @@ Void also provides proprietary video drivers, which are available in the
 | NVIDIA     | Proprietary | 500+                            | `nvidia`       |
 | NVIDIA     | Proprietary | 300/400 Series                  | `nvidia390`    |
 | NVIDIA     | Proprietary | GeForce8/9 + 100/200/300 Series | `nvidia340`    |
+
+## Input Drivers
+
+A number of input drivers are available for Xorg. If `xorg-minimal` was
+installed and a device is not responding, or behaving unexpectedly, a different
+driver may correct the issue. These drivers can grab everything from power
+buttons to mice and keyboards.
+
+| Driver                |
+|-----------------------|
+| `xf86-input-evdev`    |
+| `xf86-input-joystick` |
+| `xf86-input-libinput` |
+| `xf86-input-mtrack`   |
+| `xf86-input-synaptics`|
+| `xf86-input-vmmouse`  |
+| `xf86-input-wacom`    |
+
+## Xorg config
+
+An Xorg config for a keyboard may look something like a file
+`/etc/X11/xorg.conf.d/30-keyboard.conf` with the contents.
+
+```
+Section "InputClass"
+  Identifier "keyboard-all"
+  Driver "evdev"
+  MatchIsKeyboard "on"
+EndSection
+```
+
+> Note: For more in depth configurations, please seek other documentation.
```

Comment:
I think this is unnecessary, unless a link to documentation is given. I would lean towards just removing this though.

Reply:
removed